### ISSUE TITLE
Should live_links allow users to open in a new tab?

### ISSUE THREAD

**Issue by mylanconnolly** [phoenixframework/phoenix_live_view#528]
Good afternoon,

I have noticed a behavior with `live_link` that prevents users from doing ctrl+click (or command+click I think for macOS) to open a link in a new tab. I imagine this was by design since it wouldn't matter for a non-routed live view, however I have attempted mounting a number of live views to the router and am using live_links to navigate between them.

### Environment

* Elixir version (elixir -v):
  ```
  Erlang/OTP 22 [erts-10.6] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe]

  Elixir 1.9.4 (compiled with Erlang/OTP 21)
  ```

* Phoenix version (mix deps):

  ```
  * phoenix 1.4.11 (Hex package) (mix)
    locked at 1.4.11 (phoenix) d112c862
  ```

* Phoenix LiveView version (mix deps):

  ```
  * phoenix_live_view 0.4.1 (Hex package) (mix)
    locked at 0.4.1 (phoenix_live_view) 17e04bd0
  ```

* NodeJS version (node -v):

  ```
  v13.3.0
  ```

* NPM version (npm -v):

  ```
  6.13.1
  ```

* Operating system:

  Ubuntu 18.04 on WSL 2

### Actual behavior

When a user attempts to ctrl+click a link to open a link in a new tab, the link is instead handled in the current tab.

### Expected behavior

The user should be able to ctrl+click a link to open a link in a new tab, particularly in the case when we're using the router to link between live views.

### Some quick hacky fixes

I copied the phoenix_live_view.js file into my project and made a quick hacky change to make it work. I am sure this solution could be improved upon, but here's what I came up with:

This snippet would be located at https://github.com/phoenixframework/phoenix_live_view/blob/master/assets/js/phoenix_live_view.js#L541 in the current code base here.

```js
    window.addEventListener("click", e => {
      let target = closestPhxBinding(e.target, PHX_LIVE_LINK)
      let phxEvent = target && target.getAttribute(PHX_LIVE_LINK)
      // Begin proposed change
      if(!phxEvent || e.ctrlKey){ return }
      // End proposed change
      let href = target.href
      e.preventDefault()
      this.main.pushInternalLink(href, () => {
        Browser.pushState(phxEvent, {}, href)
        this.registerNewLocation(window.location)
      })
    }, false)
  }
```

If I'm not mistaken, macOS uses the command key for the new tab shortcut, so maybe it should check the architecture and use `e.metaKey` for macOS and `e.ctrlKey` everywhere else? I could investigate that and make a pull request if this all sounds good.

Thanks for the great project!

**Comment by snewcomer** [comment]
@mylanconnolly Great idea!  A PR would be welcome.  And you are right.  Both would be needed!

`if (!phxEvent || event.metaKey || event.ctrlKey) { return; }`

Have confirmed this works!

<img width="855" alt="Screen Shot 2019-12-13 at 2 20 21 PM" src="https://user-images.githubusercontent.com/7374640/70836000-b74f4b00-1db3-11ea-8ea8-8eabbf1fa08d.png">

**Comment by anatoliyarkhipov** [comment]
I'd also add this behavior for the middle mouse button (when you press the wheel itself).

**Comment by mylanconnolly** [comment]
Great, I'll put together a PR soon. Thanks!

**Comment by mylanconnolly** [comment]
Created a PR to add the functionality mentioned. Namely, to fall back to the browser's native handling for links where the user did the following:

- Control-click (detected by the `ctrlKey` attribute)
- Command-click (detected by the `metaKey` attribute)
- Middle-mouse click (detected by `which === 2`)

**Comment by chrismccord** [comment]
Fixed via #533
